packages/@okta/vuepress-site/code/oie/index.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Identity Engine SDKs & Samples
+---
+
+# Identity Engine SDKs & Samples
+
+<ApiLifecycle access="ie" />
+
+## What's new
+
+[Identity Engine](/docs/concepts/ie-intro/) introduces new identity management capabilities with greater flexibility and customization. To take advantage of these new features and for a better development experience, use the Identity Experience (IDX) SDKs to manage authentication in your apps. These new SDKs are recommended for orgs using the Identity Engine.
```

Comment:
IDX or IdX is an internal-use abbreviation only AFAIK??
Looks like everywhere else but just this one instance you use just "Identity Engine" not "the Identity Engine". So, just fixing this one.

```suggestion
[Identity Engine](/docs/concepts/ie-intro/) introduces new identity management capabilities with greater flexibility and customization. To take advantage of these new features and for a better development experience, use the Identity Experience SDKs to manage authentication in your apps. These new SDKs are recommended for orgs using Identity Engine.
```

Reply:
Thanks. Depending on what PM says, I might have to remove all IDX references in names of the GitHub links.